storage/sql/migrations/updates/20190214154800_remove_trs_dependant_tables.sql

```diff
@@ -14,17 +14,10 @@
 
 
 /*
-  DESCRIPTION: Counts duplicate delegates by transactionId.
-
-  PARAMETERS: None
+ * DESCRIPTION: Remove all trs dependant tables into trs asset field (transfer, signatures, delegates, votes, multisignatures, dapps, intransfer, outtransfer)
+ * Note that it will also drop the `full_blocks_list` view as it uses some of these tables
+ *
+ * PARAMETERS: None
 */
 
-WITH duplicates AS
-  (
-    SELECT count(*)
-    FROM delegates
-    GROUP BY "transactionId"
-    HAVING count(*) > 1
-  )
-SELECT count(*)
-FROM duplicates
+DROP TABLE "transfer", "signatures", "delegates", "votes", "multisignatures", "dapps", "intransfer", "outtransfer" CASCADE;
```

Comment:
In such cases you can use `git rm` to mark some file is removed. Then git will not try to link up two files as rename.